nym-vpn-core/crates/nym-vpn-account-controller/src/commander.rs

```diff
@@ -190,10 +190,10 @@ impl AccountControllerCommander {
             .get_available_tickets()
             .await
             .map_err(|err| RequestZkNymError::CredentialStorage(err.to_string()))?
-            .is_all_ticket_types_above_threshold(0)
+            .is_all_ticket_types_above_soft_threshold()
```

Comment:
This is more aggressive than the 1 threshold we mentioned, right? I see this is above 30 tickets as threshold

Reply:
Yep, if below 30 tickets then request more before trying to connect, so we have comfortably above the minimum number of tickets to operate